docs/release/trg-0/trg-1-05.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TRG 1.05 - Documentation Architecture
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft - MISSING EXTERNAL INFORMATION  | 10-Jan-2024  | Transfered from Release Acceptance Criteria into TRG|
+
+## Why
+
+It makes it much easier for others to use the component.
+>Who is the audience/adressant of the document?
+>```docs``` Folder
+
+## Description
+
+Arc42 documentation completed for the relevant product version.
```

Comment:
What does "completed" mean? Is there tangible quality criteria for Arc42 docs?

Reply:
line 22: In order to pragmatically start or update your Architecture documentation, consult ARC42 (https://arc42.org/). An "Architecture" folder must be present in the /docs/ folder of your leading repository. Within the Architecture folder the structure of the ARC42 template shall be applied. Follow the guidance to understand, WHAT and HOW to document.